docs/aqd.rst

```diff
@@ -0,0 +1,283 @@
+Interacting with smart contracts on the command line
+====================================================
+Solang Aqd (عَقد, meaning "contract" in Arabic), is a Command-Line Interface (CLI) tool 
+designed for easy interaction with smart contracts on Solana and Polkadot blockchains.
+It simplifies the process of deploying smart contracts and executing specific functions on these contracts. 
+This documentation provides essential guidance for using ``solang-aqd`` effectively.
+
+
+Installation
+____________
+
+To install `solang-aqd`, run this command:
+
+.. code-block:: bash
+
+    cargo install --force --locked aqd
+    
+To update to the latest version, use the same command.
+
```

Comment:
Could mention the feature flags for disabling targets the user is not interested in.